cypress/integration/artist.spec.js

```diff
@@ -0,0 +1,24 @@
+import { artworkGridRenders } from "../helpers/artworkGridRenders"
+
+describe("/artist/:id", () => {
+  before(() => {
+    cy.visit("/artist/pablo-picasso")
+  })
```

Comment:
These `before` clauses will result in a full network request for each `it` test case, correct?

(Seems fine for now, but potentially something to optimize if the duration of the Cypress suite becomes a concern?)

Reply:
These will use one request at the beginning of the suite, all `it` clauses will be referring to the same request. (`beforeEach` would make a new request for each test).

Reply:
Ah perfect, thanks